### PR TITLE
develop

### PR DESCRIPTION
**Pull request by siamekanto19**
Added the necessary features and components.

### CODE REVIEW

File: src/components/Tag.tsx [hunk 1]
```diff
@@ -1,5 +1,94 @@
...
+      return 'rounded-md'
+    } else {
+      return 'rounded-full'
+    }
+  }
+
+  function BGColor() {
+    if (Background === 'dark') {
+      return 'bg-gray-800 text-white'
+    } else {
```
Comment (aukgit) [inline src/components/Tag.tsx:38]: else is not required here.

File: src/components/Button.tsx [hunk 2]
```diff
@@ -1,5 +1,82 @@
...
+    }
+  }
+  function SpinnerSize() {
+    if (Size === 'sm') {
+      return 'h-4'
+    } else if (Size === 'lg') {
+      return 'h-5'
+    } else if (Size === 'xl') {
+      return 'h-6'
+    } else {
```
Comment (aukgit) [inline src/components/Button.tsx:61]: else not required

**Review by aukgit — COMMENTED** [review]
suggestions!

File: src/components/Tag.tsx (hunk 1 above)
Comment (siamekanto19) [reply src/components/Tag.tsx:38]: Thank you so much for your suggestion. I will definitely keep this in mind in the future whenever I work on some similar functionality.

File: src/components/Button.tsx (hunk 2 above)
Comment (siamekanto19) [reply src/components/Button.tsx:61]: Thank you so much for your suggestion. I will definitely keep this in mind in the future whenever I work on some similar functionality.

**Pull request merged**